src/inspector_agent.cc

```diff
@@ -865,6 +888,13 @@ void Agent::Stop() {
   impl->Stop();
 }
 
+void Agent::ContextCreated(const node::inspector::ContextInfo* info) {
+  impl->ContextCreated(info);
+}
+void Agent::ContextDestroyed(v8::Local<v8::Context> context) {
+  impl->ContextDestroyed(context);
+}
```

Comment:
in order to pass things to the right place and get through all the levels of abstraction, yes.